### DISCUSSION TITLE
E-mail to Customer for Comment and Status change in the Ticket

### DISCUSSION THREAD

**Discussion by RobertoVeca** [polonel/trudesk#506] [Ideas]
Hi there, this tool can be easily the best free Ticketing System I’ve ever used.

The really only very important (essential IMVHO) feature that’s missing, is the e-mail notification for updates (comments) and status changes of the Ticket for the Client!

Many Client does not (and won’t indeed) use the Ticketing Portal, no matter which one, they like e-mail (clients…). So again, IMVHO, for a Ticketing system is mandatory a way to send an automatic notification to the Client for any Ticket updates.

There is a possibility that this feature will come soon? Trust me, if it will, this tool will beat all the others very easily!!

**Comment by polonel** [comment]
Currently, it does send out emails but not multiple emails on every update. The mailer system is dated and is next on the list to be completely revamped and updated.

**Reply by suprematis** [reply]
Hi @polonel any update on this?

**Reply by iitbmanish** [reply]
@RobertoVeca can you share how to configure email. i am not getting emails for even ticket creation.
